tests/test_batch/test_batch_jobs.py

```diff
@@ -166,7 +166,7 @@ def test_list_jobs():
         job.should.have.key("createdAt")
         job.should.have.key("jobDefinition")
         job.should.have.key("jobName")
-        job.should.have.key("status").equals("STARTING")
+        job.should.have.key("status").equals("RUNNABLE")
```

Comment:
It seems like my change introduces a slight lag in starting the jobs. This test seems a little brittle since it is effected by timing but I see no reason why `RUNNABLE` is invalid here